tests/integration/api/test_search_format.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import requests
+from urllib.parse import quote
+from .constants import API_URL
+
+@pytest.mark.parametrize("endpoint", [
+    ("/search?query=keyword:NASA&filter=format:readable")
+])
+def test_readable_format_flags(endpoint):
```

Comment:
I think what we are testing here is that the search format filter is working as expected. I recommend changing the test name to reflect that.